Objective-C/CBLReplicator.mm

```diff
@@ -808,7 +798,12 @@ - (bool) filterDocument: (C4CollectionSpec)c4spec
                                                             kCBLDefaultCollectionName)];
     }
     
-    auto doc = [[CBLDocument alloc] initWithCollection: collection
+    NSString* name = slice2string(c4spec.name);
```

Comment:
Remove lines 784 - 800 above as no use:

https://github.com/couchbase/couchbase-lite-ios/pull/2995/commits/8f663ac9d133262a568f3fb353cd0f624668e7b5#diff-db9ac2c53f3d61c425b37311410aa8be9d6fcafb43e203cd6235c63b18455b0bR784-R800

Reply:
I thought I removed it